src/routes/Async/modules/async.js

```diff
@@ -0,0 +1,57 @@
+import fetch from 'isomorphic-fetch'
+
+// ------------------------------------
+// Constants
+// ------------------------------------
+export const FETCHED_DATA = 'FETCHED_DATA'
+export const FETCHING_DATA = 'FETCHING_DATA'
+
+// ------------------------------------
+// Actions
+// ------------------------------------
+export function startFetching () {
+  return {
+    type: FETCHING_DATA
+  }
+}
+
+export function fetchedData (data) {
+  return {
+    type: FETCHED_DATA,
+    payload: data
+  }
+}
+
+export const fetchData = () => {
+  return (dispatch) => {
+    dispatch(startFetching())
+
+    return fetch('https://jsonplaceholder.typicode.com/posts/1')
+      .then(response => response.json())
```

Comment:
We can get rid of this but I personally found the example valuable. Fetch is already built into Chrome and is very easy to get started with.